app/views/layouts/_themes.html.haml

```diff
@@ -0,0 +1,14 @@
+%ul.register-themes
+  - @registers_collection.each do |category|
+    - if category.slug == 'digital-data-and-technology-profession-capability-framework' and current_page?(root_path)
```

Comment:
can we move this out of the template and into the controller or model?

Reply:
I think we could query: `Category.where.not(slug: 'digital-data-and-technology-profession-capability-framework')`

Reply:
Logic now resides in model - can you give it a check to see if you're happy with it?